packages/commonwealth/client/scripts/helpers/index.tsx

```diff
@@ -174,16 +178,16 @@ export function renderMultilineText(text: string) {
  * blocknum helpers
  */
 
-export function blocknumToTime(blocknum: number): moment.Moment {
+export function blocknumToTime(app: IApp, blocknum: number): moment.Moment {
```

Comment:
lets just only pass the props it needs -> `block: IBlockInfo`. We are trying to minimize and completely remove the app object.

Reply:
Good idea, fixed.